modules/form-field/react/stories/stories_ColorInput.tsx

```diff
@@ -124,3 +150,61 @@ storiesOf('Components|Inputs/Color Picker/Color Input/React/Left Label', module)
       {controlComponent(<ColorInput />)}
     </FormField>
   ));
+
+storiesOf('Components|Inputs/Color Picker/Color Input/React/Visual Testing', module)
+  .addParameters({component: ColorInput})
+  .addDecorator(withReadme(README))
+  .add('States', () => (
+    <StaticStates>
```

Comment:
Do you mind updating this with the changes in #418?

Reply:
Updated.